hikari/impl/rest.py

```diff
@@ -759,6 +765,23 @@ async def _request(
                     real_url = str(response.real_url)
                     raise errors.HTTPError(f"Expected JSON [{response.content_type=}, {real_url=}]")
 
+                # Handling 5xx errors
+                if 500 <= response.status < 600 and retries_done <= self._max_retries and self._max_retries != 0:
+                    if backoff is None:
+                        backoff = rate_limits.ExponentialBackOff()
```

Comment:
Having some default values here will help not have a huge backoff

Reply:
I don't understand, could you please explain @davfsa ?